core/cat/routes/upload.py

```diff
@@ -54,6 +54,28 @@ async def upload_file(
         "info": "File is being ingested asynchronously",
     }
 
+# receive chunk via http endpoint
+@router.post("/chunk")
+async def upload_chunk(
+    request: Request,
+    background_tasks: BackgroundTasks,
+    chunk: str = Body(
+        description="A chunk to upload in the cat memory",
+        examples={"Example of chunk with length greater than 20"},
+        min_length=20
```

Comment:
is there any particular reason for the choice of 20?

Reply:
no particular reason, what value is best to have? 10 like [here](https://github.com/cheshire-cat-ai/core/blob/4c9fff02470ecd406a346652d035aa98ada71d58/core/cat/rabbit_hole.py#L418C9-L419C1) ?